src/encode.rs

```diff
@@ -359,8 +360,7 @@ impl Encoder {
   /// 
   /// Raw ffmpeg encoder belonging to given stream.
   fn encoder(writer_stream: &StreamMut) -> Result<AvEncoder> {
-    writer_stream
-      .codec()
+    Context::from_parameters(writer_stream.parameters())?
```

Comment:
As mentioned in https://github.com/zmwangx/rust-ffmpeg/blob/feae6b21b8c9f76d399260f812c0deb620bf02ad/CHANGELOG.md